R/Reactant/build_tarballs.jl

```diff
@@ -439,7 +434,15 @@ for gpu in ("none", "cuda"), mode in ("opt", "dbg"), cuda_version in ("none", "1
     end
 
     should_build_platform(triplet(augmented_platform)) || continue
-    products2 = copy(products)
+	
+    # The products that we will ensure are always built
+    products2 = Product[]
+    if Sys.isapple(platform)
+	push!(products2, LibraryProduct(["libReactantExtra", "libReactantExtra"], :libReactantExtra))
+    else
+	push!(products2, LibraryProduct(["libReactantExtra", "libReactantExtra"], :libReactantExtra, dlopen_flags=[:RTLD_GLOBAL,:RTLD_DEEPBIND]))
```

Comment:
Can you elaborate on why `RTLD_GLOBAL` helps?  Also, `RTLD_GLOBAL` is a bit problematic because it makes symbols available to everyone, and this library ships loads of symbols.

Reply:
It’s needed for Linux with precompiled, since the precompiled so can’t find the symbol otherwise in my PR. I confirmed loading like this fixes locally